.github/workflows/unit-tests.yml

```diff
@@ -22,18 +22,40 @@ jobs:
 
     steps:
 
-      - uses: actions/setup-python@v2
+      - name: Setup Python
+        uses: actions/setup-python@v2
         with:
           python-version: ${{ matrix.python-version }}
-      - run: python -m pip install --upgrade pip
 
-      - uses: actions/checkout@v2
+      - name: Upgrade pip
+        run: python -m pip install --upgrade pip
 
-      - run: python -m pip install .[test]
+      - name: Checkout code
+        uses: actions/checkout@v2
 
-      - run: python -m pytest tests --cov=endaq --hypothesis-profile=ci
+      - name: Get pip Cache Location
+        id: pip-cache
+        run: echo "::set-output name=dir::$(pip cache dir)"
 
-      - uses: codecov/codecov-action@v2
+      - name: Get date
+        id: date
+        shell: bash
+        run: echo "::set-output name=text::$(date +%Y/%m)"
```

Comment:
```suggestion
        run: python -c "import datetime; now = datetime.datetime.now(); print(f'::set-output name=text::{now.year}-{now.month}-{now.day % 7}')"
```

Reply:
...oh yeah we have Python 🤣 good suggestion, thanks

Reply:
I changed it to be every 8 days, so that it doesn't reset after a single day at the end of February, which only has 29 days.

Reply:
lmao I did a modulus.  new changes look good